site/404.html

```diff
@@ -179,6 +179,9 @@ <h2 id="oops">Oops!</h2>
         <h5>Latest News</h5>
         <ul class="list-unstyled">
           
+          <li><a href="/news/spark-4.0.0-preview2.html">Preview release of Spark 4.0</a>
```

Comment:
The link is correct but the link text "Preview release of Spark 4.0" is the same as preview 1. Do you think we can make them different?

Reply:
Ya, I also felt in that way, but simply followed our convention at 3.0.0-preview2 here.
![Screenshot 2024-09-26 at 09 23 48](https://github.com/user-attachments/assets/6fbdea83-6339-4f24-8a13-541088ff50ed)

Reply:
It seems that we kept `New title itself` in the same way.

Reply:
Okay.